packages/design-picker/src/components/style.scss

```diff
@@ -231,8 +231,6 @@
 	.design-picker__pricing-description {
 		text-align: left;
 		color: #50575e;
-		z-index: 1;
```

Comment:
tested and the link is still clickable!

Reply:
good point I actually hadn't thought to test that 😅